CMakeLists.txt

```diff
@@ -356,6 +356,7 @@ endif()
 target_compile_definitions(sdl-build-options INTERFACE "-DUSING_GENERATED_CONFIG_H")
 target_include_directories(sdl-build-options
   INTERFACE
+    "${SDL3_BINARY_DIR}/include-config-$<LOWER_CASE:$<CONFIG>>/SDL3"
```

Comment:
Why do we need this?

Reply:
We don't. Removed it. Thanks for spotting this.